scripts/validate-license.sh

```diff
@@ -15,18 +15,13 @@
 # limitations under the License.
 set -euo pipefail
 IFS=$'\n\t'
-
+# removed -not to reduce complexity
```

Comment:
This line can probably be removed since it is only applicable right now since `-not` is part of the existing `validate-license.sh` script.